### PR TITLE
Description was somewhat overly broad

### PR DESCRIPTION
**Pull request by danbri**
From discussion at W3C TPAC event @lehors suggested that this example sets over-broad expectations for the project.

Context: http://www.w3.org/2015/10/28-amp-irc#T05-26-05

### CODE REVIEW

**Comment by danbri** [comment]
I work for Google and I'm in the /google/ Github organization, so I am surprised to be asked for CLA (although I've just signed one anyway). "I signed it!" for @googlebot ...

File: docs/create_page.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ The basic AMP HTML page includes the following mark-up:
       {
         "@context": "http://schema.org",
         "@type": "NewsArticle",
-        "headline": "Open-source framework for publishing content everywhere",
+        "headline": "Open-source framework for publishing content",
```
Comment (dvoytenko) [inline docs/create_page.md]: I think this meant to mean that this is a solution not bound to any particular platform.

Comment (jmadler) [inline docs/create_page.md]: Agree, but that can be covered later in the tutorial if needed.

**Comment by jmadler** [comment]
LGTM - deferring to tech lead for final review

**Comment by cramforce** [comment]
LOLTM^WLGTM

**Comment by danbri** [comment]
Thanks!

**Pull request merged**